lib/fake_artist/game.ex

```diff
@@ -0,0 +1,98 @@
+defmodule FakeArtist.Game do
+  use GenStateMachine
+  require Logger
+
+  @spec start_link(list()) :: tuple()
+  def start_link(players) do
+    {game_master, players_without_game_master} = get_random_player(players)
+    {trickster, players_without_roles} = get_random_player(players_without_game_master)
+    roles = get_roles(game_master, trickster, players_without_roles)
+    seats = get_seats(game_master, players_without_game_master)
+    active_seat = 0
```

Comment:
all of this logic should probably happen in an [`init/1` callback](https://hexdocs.pm/gen_state_machine/GenStateMachine.html#c:init/1) and this fn can just be:
```elixir
def start_link(players) do
  GenStateMachine.start_link(__MODULE__, players)
end
```
that follows the typical genserver pattern of "light in the frontend, party in the backend"

Reply:
Ya I think that's better.